### ISSUE TITLE
Fix docker sanity check for ansible 2.8+

### ISSUE THREAD

**Issue by znerol** [ansible/molecule#2053]
# Issue Type

- Bug report

# Desired Behavior

Travis build passes for ansible 2.8

# Actual Behaviour

Even after fixing the `testinfra` dependency the 2.8 build still does not pass on travis. This is due to the fact that `test_sanity_checks_missing_docker_dependency` tries to import a module which was moved in ansible 2.8.

From a recent build log: https://travis-ci.com/ansible/molecule/jobs/201279446#L1163

```
=================================== FAILURES ===================================
_________________ test_sanity_checks_missing_docker_dependency _________________
thing = <module 'ansible.module_utils' from '/home/travis/build/ansible/molecule/.tox/ansible28-unit/lib/python3.7/site-packages/ansible/module_utils/__init__.py'>
comp = 'docker_common', import_path = 'ansible.module_utils.docker_common'
    def _dot_lookup(thing, comp, import_path):
        try:
>           return getattr(thing, comp)
E           AttributeError: module 'ansible.module_utils' has no attribute 'docker_common'
/opt/python/3.7.1/lib/python3.7/unittest/mock.py:1083: AttributeError
During handling of the above exception, another exception occurred:
mocker = <pytest_mock.MockFixture object at 0x7f913a7cee80>
_instance = <molecule.driver.docker.Docker object at 0x7f913bbe4780>
    def test_sanity_checks_missing_docker_dependency(mocker, _instance):
        target = 'ansible.module_utils.docker_common.HAS_DOCKER_PY'
>       mocker.patch(target, False)
/home/travis/build/ansible/molecule/test/unit/driver/test_docker.py:147: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/home/travis/build/ansible/molecule/.tox/ansible28-unit/lib/python3.7/site-packages/pytest_mock.py:158: in __call__
    return self._start_patch(self.mock_module.patch, *args, **kwargs)
/home/travis/build/ansible/molecule/.tox/ansible28-unit/lib/python3.7/site-packages/pytest_mock.py:138: in _start_patch
    mocked = p.start()
/opt/python/3.7.1/lib/python3.7/unittest/mock.py:1386: in start
    result = self.__enter__()
/opt/python/3.7.1/lib/python3.7/unittest/mock.py:1239: in __enter__
    self.target = self.getter()
/opt/python/3.7.1/lib/python3.7/unittest/mock.py:1409: in <lambda>
    getter = lambda: _importer(target)
/opt/python/3.7.1/lib/python3.7/unittest/mock.py:1096: in _importer
    thing = _dot_lookup(thing, comp, import_path)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
thing = <module 'ansible.module_utils' from '/home/travis/build/ansible/molecule/.tox/ansible28-unit/lib/python3.7/site-packages/ansible/module_utils/__init__.py'>
comp = 'docker_common', import_path = 'ansible.module_utils.docker_common'
    def _dot_lookup(thing, comp, import_path):
        try:
            return getattr(thing, comp)
        except AttributeError:
>           __import__(import_path)
E           ModuleNotFoundError: No module named 'ansible.module_utils.docker_common'
/opt/python/3.7.1/lib/python3.7/unittest/mock.py:1085: ModuleNotFoundError
---------------------------- Captured stdout setup -----------------------------
```

In order to resolve the issue the changes from #1985 need to be replicated for the test code as well.

**Comment by decentral1se** [comment]
See https://github.com/ansible/molecule/pull/2050.

**Comment by znerol** [comment]
Fixed via #2050
